model_analyzer/model_manager.py

```diff
@@ -55,6 +56,13 @@ def __init__(self, config, gpus, client, server, metrics_manager,
         self._result_manager = result_manager
         self._state_manager = state_manager
 
+        if state_manager.starting_fresh_run():
```

Comment:
Is there a required ordering between this and the setting of mvnm a few lines lower?

Reply:
Yes